container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/logs_collector_functions/shared_helpers.go

```diff
@@ -13,12 +13,13 @@ import (
 	"github.com/kurtosis-tech/stacktrace"
 	v1 "k8s.io/api/apps/v1"
 	apiv1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
 	"net"
 	"time"
 )
 
 const (
-	maxAvailabilityCheckRetries     = 30
+	maxRetries                      = 30
```

Comment:
My spidey sense is going off, saying that this _might_ be [over-DRY'ing](https://github.com/kurtosis-tech/style-guide?tab=readme-ov-file#dont-over-dry-your-code-and-create-connections-between-components-when-none-are-needed) code. I.e. that you have at least two separate semantic values (number of retries on waiting for a namespace to get removed, number of retries on waiting for availability), that are now incorrectly getting squashed into one value. 

However, I'm not positive on this - just suggesting it for consideration

Reply:
separated these values